rainforest-cli.go

```diff
@@ -8,25 +8,33 @@ import (
 
 var apiToken string
```

Comment:
These variable lists look a lot cleaner if you do multiple assignment, e.g.:
```go
var (
  foo int
  bar = "hello"
)
```
etc.